vector/README.md

```diff
@@ -105,6 +105,26 @@ $ go test -v ./vector/...
 Push your changes to `VECTOR_MANIFEST` up execution in CI.
 
 
+## Running vector benchmarks
+
+The vectors can be executed as part of a Go benchmark suite. The benchmarks output numbers of blockstore gets per operation as a custom metric.
+
+Due to the length of time needed to execute, it is recommended to supply a custom benchtime for running each benchmark and an overall timeout for the test run that is greater than the expected benchtime for all benchmarks.
+
+To run benchmarks for all the vectors
+
+```
+go test -v -bench=BenchmarkExecuteVectors/.* -benchtime 1m -timeout 30m  ./vector/...
+```
+
+To run benchmarks for a single vector:
+
+```
+go test -v -bench=BenchmarkExecuteVectors/QmYhszKDYDFvXyFQF8KefF4M8gcNaoTC8FMYFcgKmfq6pi_miner.* -benchtime 1m -timeout 30m  ./vector/...
```

Comment:
Does this work despite the test names being derived from the vector file names?

Reply:
Yes. Would be nicer if we could give vectors short names though.

Reply:
Agreed, these names are obnoxious. 

Idea: We could replace the hash with a number (to ensure uniqueness) iff we maintain the ordering in the `VECTOR_MANIFEST` file.

Reply:
I'm a bit wary about just using ordering since it could change and then automated benchmarking would be out of step between runs. I will look at maybe stripping the hash from the front in the test name.

Reply:
> I will look at maybe stripping the hash from the front in the test name.

I think that should work fine as long as we are careful when we write the vector name to avoid collisions.

Reply:
Updated to use nicer names